pkgs/applications/networking/instant-messengers/bitlbee/plugins.nix

```diff
@@ -1,20 +1,15 @@
-{ lib, stdenv, bitlbee }:
+{ lib, runCommandLocal, bitlbee }:
 
 with lib;
 
-plugins:
-
-stdenv.mkDerivation {
-  inherit bitlbee plugins;
-  name = "bitlbee-plugins";
+plugins: runCommandLocal "bitlbee-plugins"
+{
```

Comment:
```suggestion
{
  inherit plugins;
```

Reply:
also, `{` shouldn't have a newline in the front
https://nixos.org/manual/nixpkgs/unstable/#sec-syntax

Reply:
good idea. should be now as expected.